tests/test_printer.py

```diff
@@ -32,22 +32,6 @@ class Query:
     assert print_schema(schema) == textwrap.dedent(expected_type).strip()
 
 
-# def test_recursive_type():
```

Comment:
same for this one :)

Reply:
😆 Ok, I will restore it.

Reply:
This test is broken right now. I will try to fix it but not sure if I will be able with my inexistent background using 🍓.

Reply:
The test was created here: https://github.com/strawberry-graphql/strawberry/commit/d37cdac746cb000e43e19d82d42e2aa8d9a58b61
Never ran. :D

Reply:
let's delete it then ;)